src/EnergyPlus/api/func.cc

```diff
@@ -88,68 +88,68 @@ void registerErrorCallback(EnergyPlusState state, void (*f)(int, const char *))
 Glycol glycolNew(EnergyPlusState state, const char *glycolName)
 {
     auto *thisState = reinterpret_cast<EnergyPlus::EnergyPlusData *>(state);
-    auto *glycol = new EnergyPlus::FluidProperties::GlycolAPI(*thisState, glycolName);
+    auto *glycol = EnergyPlus::Fluid::GetGlycol(*thisState, EnergyPlus::Util::makeUPPER(glycolName));
```

Comment:
This call used to go through a `GlycolAPI` object which held the `GlycolName` and `GlycolIndex` fields that were used by the old procedural API. We now have an object-oriented API that doesn't use those fields and so the internal `GlycolProps *` can be directly cast to the extenrnal `Glycol` type handle without the need for an intermediate object.